microprofile-jwt/src/main/java/org/jboss/eap/qe/microprofile/jwt/tools/JwtHelper.java

```diff
@@ -0,0 +1,63 @@
+package org.jboss.eap.qe.microprofile.jwt.tools;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.Signature;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.UUID;
+
+/**
+ * Sugar syntax for generating JWTs with re-used values
+ */
+public final class JwtHelper {
```

Comment:
again :) Why do we need this file?

Isn't available any standard class for this ?

How will customers do this stuff ? If each of them will do the same WF/EAP should provide it for them.

Reply:
https://github.com/jboss-eap-qe/eap-microprofile-test-suite/pull/38#discussion_r350114220